.scrutinizer.yml

```diff
@@ -1,6 +1,6 @@
 filter:
     paths: [PhpAmqpLib/*]
-    excluded_paths: [tests/*, demo/*, benchmark/*, spec/*]
+    excluded_paths: [tests/*, demo/*, benchmark/*, spec/*, PhpAmqpLib/Helper/Protocol/*]
```

Comment:
This is because these files are generated and we have little control over how they are formatted.